internal/app/networkd/pkg/address/static.go

```diff
@@ -9,15 +9,16 @@ import (
 	"net"
 	"time"
 
-	"golang.org/x/sys/unix"
-
 	"github.com/talos-systems/talos/pkg/config/machine"
+	"golang.org/x/sys/unix"
```

Comment:
our convention is to keep talos imports as a group

Reply:
FYI. `make fmt` should fix this